landing-pages/site/content/en/ecosystem/_index.md

```diff
@@ -31,6 +31,8 @@ If you would you like to be included on this page, please reach out to the [Apac
 
 [Apache Airflow: Complete Hands-On Beginner to Advanced Class](https://www.udemy.com/course/apache-airflow-course) by Alexandra Abbas on Udemy
 
+[Data Pipelines with Apache Airflow](https://www.manning.com/books/data-pipelines-with-apache-airflow) Apache Airflow Book by Bas P. Harenslak and Julian Rutger de Ruiter on Amazon
```

Comment:
BTW. I believe there is a second edition of that book being written as we speak by different authors, so maybe leaving the details out about the authorship might be a good idea.

Reply:
@potiuk I've removed the authorship details for now .